src/kOS/SharedObjects.cs

```diff
@@ -12,15 +12,27 @@ public class SharedObjects : Safe.SafeSharedObjects
         public ProcessorManager ProcessorMgr { get; set; }
         public Part KSPPart { get; set; }
         public TermWindow Window { get; set; }
+        public List<KOSManagedWindow> ManagedWindows { get; }
```

Comment:
The `TermWindow` would also technically qualify.  This list should also be cleared, and the contained windows destroyed with `UnityEngine.Object.Destroy`, in the `DestroyObjects` method, unless they have no reference to `SharedObjets` to create a circular reference.